next/lib/graphql/fragments/nodes.ts

```diff
@@ -102,6 +103,15 @@ export const FRAGMENT_NODE_PAGE = graphql(`
   }
 `);
 
+export const FRAGMENT_NODE_TESTCONTENT = graphql(`
+  fragment FragmentNodeTestcontent on NodeTestContent {
+    body {
+      ...FragmentTextSummary
+    }
+    testfield
+  }
+`);
+
```

Comment:
Here we create a new fragment adding the fields that we are interested in for this specific content types. Note that title, id, metatags etc are not needed, because we will add this fragment to the FRAGMENT_NODE_UNION above, so those fields will be available automatically.